app/domain/authentication/authn_k8s/inject_client_cert.rb

```diff
@@ -116,11 +124,7 @@ def common_name
         @common_name ||= CommonName.new(smart_csr.common_name)
       end
 
-      def validate_cert_installation(resp)
-        error_stream = resp[:error]
-        return if error_stream.nil? || error_stream.empty?
-        raise Err::CertInstallationError, cert_error(error_stream)
-      end
+
 
```

Comment:
looks like these lines are redundant. verify that you have only one whitespace line